File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int slow = 1;
        int fast = nums[0] ;
        for(int i =1; i<nums.length; i++)
        {
            if(nums[i] != fast)
            {
                nums[slow] = nums[i];
                fast = nums[slow];
                slow++;
            }
        }
        return slow;
    }
}

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& arr) {
        int count =1;
        int start=0,end=1;
        while(end<arr.size()){
            if(arr[start]!=arr[end]){
                count++;
                start++;
                arr[start]=arr[end];
                end++;
            }
            else{
                end++;
            }
        }
        return count;
    }
};